server/routes/daily-attendance-summary/handlers/attendance.test.ts

```diff
@@ -0,0 +1,392 @@
+
+      when(prisonService.searchInmatesByPrisonerNumbers)
+        .calledWith(['ZXY123'], res.locals.user)
+        .mockResolvedValue(mockPrisonApiResponse)
+
+      req = {
+        query: {
+          date: dateString,
+          status: 'NotAttended',
+        },
+        session: {
+          attendanceSummaryFilters: {
+            categories: ['Education'],
+            categoryFilters: [{ value: 'Education', text: 'Education', checked: true }],
+          },
+        },
+      } as unknown as Request
+
+      await handler.GET(req, res)
+
+      expect(res.render).toHaveBeenCalledWith('pages/daily-attendance-summary/not-attended', {
+        activityDate: date,
+        status: 'NotAttended',
+        attendees: [
+          {
+            name: 'Joe Bloggs',
+            prisonerNumber: 'ABC123',
+            location: 'MDI-1-001',
+            attendance: {
+              activityId: 1,
+              activitySummary: 'Maths Level 1',
+              attendanceId: 1,
+              attendanceReasonCode: null,
+              categoryName: 'Education',
+              issuePayment: null,
+              prisonCode: 'MDI',
+              prisonerNumber: 'ABC123',
+              sessionDate: '2022-10-10',
+              status: 'WAITING',
+              timeSlot: 'AM',
+            },
+          },
+          {
+            name: 'Alan Key',
+            prisonerNumber: 'ABC321',
+            location: 'MDI-1-002',
+            attendance: {
+              activityId: 1,
+              activitySummary: 'Maths Level 1',
+              attendanceId: 2,
+              attendanceReasonCode: null,
+              categoryName: 'Education',
+              issuePayment: null,
+              prisonCode: 'MDI',
+              prisonerNumber: 'ABC321',
+              sessionDate: '2022-10-10',
+              status: 'WAITING',
+              timeSlot: 'AM',
+            },
+          },
+        ],
+        attendanceSummaryFilters: {
+          activityDate: date,
+          categoryFilters: [{ value: 'Education', text: 'Education', checked: true }],
+          activityFilters: [{ value: 'Maths Level 1', text: 'Maths Level 1', checked: true }],
+          searchTerm: '',
+        },
+      })
+    })
+  })
+
+  describe('POST', () => {
+    it('should update category filter to Education', async () => {
+      req = {
+        query: {
+          date: '2022-10-10',
+          status: 'NotAttended',
+        },
+        body: {
+          categoryFilters: ['Education'],
+        },
+        session: {
+          attendanceSummaryFilters: testAttendanceSummaryFilters(),
+        },
+      } as unknown as Request
+
+      await handler.POST(req, res)
+
+      const { attendanceSummaryFilters } = req.session
+
+      // Different from default filter values
+      expect(
+        attendanceSummaryFilters.categoryFilters.includes({ value: 'Education', text: 'Education', checked: true }),
+      )
```

Comment:
I don't think you can use `expect()` like this? I think you need to explicitly provide an expected value. Eg:

```
expect(
    attendanceSummaryFilters.categoryFilters.includes({ value: 'Education', text: 'Education', checked: true }),
).toBe(true)
```

Reply:
Expected value explicitly provided